pkg/deploy/devfile_registry.go

```diff
@@ -50,10 +52,12 @@ func SyncDevfileRegistryToCluster(deployContext *DeployContext, cheHost string)
 			domain = cheHost
 			endpoint = domain + "/" + DevfileRegistry
 		}
+
+		routePathPrefix := fmt.Sprintf("%s://%s:%d", DevfileRegistryServiceProtocol, DevfileRegistry, DevfileRegistryServicePort)
```

Comment:
This is the internal URL.

Reply:
Code was very changed after rebase with master.